packages/drivers/odsp-driver/src/odspDocumentService.ts

```diff
@@ -226,8 +226,10 @@ export class OdspDocumentService
 				service.get(from, to, telemetryProps, fetchReason),
 			// Get cachedOps Callback.
 			async (from, to) => {
-				const res = await this.opsCache?.get(from, to);
-				return (res as ISequencedDocumentMessage[]) ?? [];
+				if (this.odspResolvedUrl.fileVersion !== undefined) {
```

Comment:
its a bit more efficient to pull the condition out of the function, and instead create the function based on the condition:
` this.odspResolvedUrl.fileVersion !== undefined ? ()=>[] : ...`

also, can you add a todo comment with the work item number here, so its easy to track that we plan to fix this.